src/main/java/org/takes/rs/RsVelocity.java

```diff
@@ -168,7 +169,7 @@ private static InputStream render(final InputStream page,
      * Pair of values.
      */
     public static final class Pair
-        extends AbstractMap.SimpleEntry<String, Object> {
+        extends AbstractMap.SimpleEntry<CharSequence, Object> {
```

Comment:
Ditto. Shouldn't it be `? extends CharSequence` just like in `RsWithHeaders` constructor?